### DISCUSSION TITLE
Slower speeds then Ookla & Fast.com

### DISCUSSION THREAD

**Discussion by DenisDee** [alexjustesen/speedtest-tracker#369] [Q&A]
I recently installed speedtest-tracker and i always notice that speeds shown in script are lower then what I get in Ookla speedtest and fast.com. Speedtest-tracker shows me 150-300mbps where in same time on other two I get real speed from provider, 600-1000mbps.

**Comment by alexjustesen** [comment]
Are you running speedtest tracker on windows?

**Reply by DenisDee** [reply]
No, I run it on docker on Raspberry Pi.

**Comment by alexdelprete** [comment]
> Speedtest-tracker shows me 150-300mbps where in same time on other two I get real speed from provider, 600-1000mbps.

Make sure the same server ID is used. You can see from detailed view of the results which server has been used by speedtest-tracker.

Once you know the server ID used by ST, use speedtest-cli from other machines with -s parameter and ID of the server used by ST. This is the only way you can be sure the same server is used.

**Reply by DenisDee** [reply]
I will give it a try tomorrow when I get in studio and will report back.

**Comment by daendir** [comment]
I had the same problem. I have found that only the raspberry pi 4 has native 1000 ethernet. The raspberry pi 3 b+ and below uses the ethernet controller through the usb ports, which means that it does not have enough width to transmit more than 200 mbps.
